setup.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+from setuptools import setup, find_packages, find_namespace_packages
+
+setup(
+    name='ensembl_metadata_registry',
+    version='0.1',
+    packages=find_namespace_packages(),
```

Comment:
```suggestion
    packages=find_packages(),
```
As packages have `__init__.py` files.